tests/system/hw_math/source/main.c

```diff
@@ -49,6 +125,9 @@ int main(int argc, char **argv)
     printf("qNaN: 0x%08lX\n", quiet_nan);
     printf("\n");
 
+    volatile float* vf=malloc(sizeof(float));
+    if (vf==0) printf("could not allocate volatile float!\n");
```

Comment:
This isn't needed. All you need to do is put `volatile float vf;` as a global variable and write to it from below.

Reply:
I'll remove the Malloc call and perhaps also replace TIMER3_CR since it's also a pointer.

Reply:
Yeah, sorry, that's what I meant. You can use the global in all the loops.